### ISSUE TITLE
Better DB Schema

### ISSUE THREAD

**Issue by softbobo** [wuest-halle/wuestbot#33]
Already now it becomes apparent, that the quickly constructed DB schema will not satisfy all our (future) needs. For example in #27 we state, that sending the location alongside the event was a nice feature. That makes a new table necessary. What is necessary to optimize our DB structure? I think:
* reconsidering our choices for the storage of dates and times
* putting constraints on text columns
* using an ORM - here `SQLAlchemy` is feasible, as mentioned elsewhere
* introducing a location table

Also, at the moment our DB is very thin and hasn't got a lot of realations going on. Yet we should at least discuss the possibility of using a different DBMS that might suit our future needs better. 

The whole proposal is to be seen as part of a bigger milestone, probably 1.0.0

**Comment by obitech** [comment]
We should also look at enabling [database migrations](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-iv-databas) while we're at it

**Comment by softbobo** [comment]
I agree on database migrations. I think it is also advisable to think about an automated backup mechanism of our data. I know that `SQLite` has something like this [built in](https://www.sqlite.org/c3ref/backup_finish.html#sqlite3backupinit) but am not sure how this is implemented in `SQLAlchemy`, if at all.

**Comment by obitech** [comment]
> I think it is also advisable to think about an automated backup mechanism of our data.

I agree. But that is probably more like an "operations problem" which we need to discuss in a different step/issue as it's independent of using SQLAlchemy.
